workalendar/asia/china.py

```diff
@@ -66,13 +73,21 @@
             'Mid-Autumn Festival Shift': [(9, 18)],
             'National Day Shift': [(9, 26), (10, 9)]
         },
+    2022:
+        {
+            'Spring Festival Shift': [(1, 29), (1, 30)],
+            'Ching Ming Festival Shift': [(4, 2)],
+            'Labour Day Holiday Shift': [(4, 24), (5, 7)],
+            'National Day Shift': [(10, 8), (10, 9)]
+        },
 }
 
 
 @iso_register('CN')
 class China(ChineseNewYearCalendar):
     "China"
     # WARNING: Support 2018, 2019 currently, need update every year.
```

Comment:
we may also want to change this comment, don't you think.

Reply:
updated, thanks for your comment.